packages/nhsuk.js

```diff
@@ -2,10 +2,12 @@
 import Header from './components/header/header';
 import SkipLink from './components/skip-link/skip-link';
 import Details from './components/details/details';
+import Panel from './components/panel/panel';
```

Comment:
Can this go before SkipLink in alphabetical order for consistency

Would be good if we could put the imports in alphabetical order despite it being unrelated to this PR.

Reply:
Please could you add these changes to the documentation for ES6 imports? https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/installation/installing-with-npm.md#option-2-import-javascript-using-modules